charts/cachetarget/templates/statefulset.yaml

```diff
@@ -0,0 +1,144 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "cachetarget.fullname" . }}
+  namespace: {{ include "cachetarget.namespace" . }}
+  labels:
+    helm.sh/chart: {{ include "cachetarget.chart" . }}
+    app.kubernetes.io/name: {{ .Chart.Name }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    app.kubernetes.io/component: cache
+spec:
+  replicas: 1
+  serviceName: {{ include "cachetarget.fullname" . }}
+  selector:
+    matchLabels:
+      app: {{ include "cachetarget.fullname" . }}
+  template:
+    metadata:
+      labels:
+        app: {{ include "cachetarget.fullname" . }}
+    spec:
+      containers:
+      - name: {{ include "cachetarget.fullname" . }}
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+        env:
+          - name: PORT
+            value: "{{ .Values.ports.server }}"
+          - name: STARDOG_HOME
+            value: "/var/opt/stardog/"
+          - name: STARDOG_LICENSE_PATH
+            value: "/etc/stardog-license/stardog-license-key.bin"
+          - name: STARDOG_PROPERTIES
+            value: "/etc/stardog-conf/stardog.properties"
+          - name: STARDOG_SERVER_JAVA_ARGS
+            value: "-XX:ActiveProcessorCount={{ .Values.resources.requests.cpu }}  -Djava.io.tmpdir={{ .Values.tmpDir }} {{ .Values.javaArgs }}"
```

Comment:
`.resources` needs to be removed from `.Values.resources.requests.cpu`

Reply:
I added resources to values to match the Stardog and ZK charts instead